src/cloud-instance-docker/update-schemas.sh

```diff
@@ -0,0 +1,2 @@
+curl https://raw.githubusercontent.com/icpc/live-v3/main/schemas/advanced.schema.json >.vscode/schemas/advanced.schema.json
```

Comment:
perhaps should specify relativity to home?
**~/**.vscode/schemas/advanced.schema.json